components/sup/src/manager/mod.rs

```diff
@@ -495,7 +495,12 @@ impl Manager {
             let es_config =
                 cfg.event_stream_config
                    .expect("Config should be present if the EventStream feature is enabled");
-            let ec = EventCore::new(&es_config, &sys);
+
+            // Collect the FQDN of the running machine
+            let fqdn = habitat_core::os::net::fqdn().unwrap_or_default();
```

Comment:
Since `fqdn()` returns an `Option<String>`, `unwrap_or_default()` is going to return an empty string if `fqdn()` returns `None`... I'm not sure how useful that will be for our purposes here.

Reply:
I thought we would send an empty fqdn inside the protobuf mesasge. By sending the default string will cause the protobuf to be "empty".

Reply:
If that's the desired behavior on the Automate side, then you'd just keep it as `None`, which would ultimately give you an empty field in the protobuf. There's a difference between `None` and `Some("")`.

Reply:
Oops. In golang, an empty string field is an empty string. I can update this as you recommend.

Reply:
I chose to use hostname instead for now, I really don't know how to implement the `Option` in protobuf. 😞